src/libraries/Microsoft.Extensions.Options/src/OptionsMonitor.cs

```diff
@@ -85,8 +85,9 @@ public TOptions CurrentValue
         /// </summary>
         public virtual TOptions Get(string? name)
         {
-            name = name ?? Options.DefaultName;
```

Comment:
This technically isn't necessary because the cache API accepts null. For compat, I left this logic on the branch where we are dealing with an unknown cache implementation.